app/src/main/java/org/mozilla/fenix/settings/studies/StudiesView.kt

```diff
@@ -138,20 +173,6 @@ class StudiesView(
 
     @VisibleForTesting
     internal fun quitTheApp() {
-        Toast.makeText(
-            context,
-            context.getString(R.string.studies_toast_quit_application),
-            Toast.LENGTH_LONG
-        ).show()
-        Handler(Looper.getMainLooper()).postDelayed(
-            {
-                exitProcess(0)
-            },
-            OVERRIDE_EXIT_DELAY
-        )
-    }
-
-    companion object {
-        private const val OVERRIDE_EXIT_DELAY = 3000L
+        exitProcess(0)
```

Comment:
We no longer need the `post` for this?

Reply:
We used the `postDelayed` function to delay  killing the app `exitProcess(0)` and giving some time to the user to read the toast message :)